libcodechecker/server/client_db_access_handler.py

```diff
@@ -1243,6 +1271,10 @@ def getCheckerCounts(self, run_ids, report_filter, cmp_data):
                     # There is no difference.
                     return results
 
+            if not run_ids:
+                res = session.query(Run.id).all()
```

Comment:
Why do you query this information again? 
https://github.com/gyorb/codechecker/blob/fbc554df1732f29a82b11b1fea54687d57460980/libcodechecker/server/client_db_access_handler.py#L1258-L1260